Application/EdFi.Ods.Common/Serialization/SerializationHelper.cs

```diff
@@ -15,7 +15,7 @@ public static class SerializationHelper
     {
         public static T DeserializeFromCompressedBase64<T>(string base64EncodedString)
         {
-            if (base64EncodedString.Contains("\n"))
+            if (base64EncodedString.Contains(Environment.NewLine))
```

Comment:
This seems to have been incorrectly implemented in the first place.  If you Base64Encode a string and set formatting options to include line breaks, you can end up with "\r\n" in the output, but never "\n".  Even with .net standard on linux, for compatibility reasons it still outputs "\r\n".  See https://github.com/dotnet/runtime/issues/24704.  That aside, this method isn't used and seems to have some issues, so may just want to delete this method.